test/addons/rook-pool/storage-class.yaml

```diff
@@ -6,6 +6,8 @@ apiVersion: storage.k8s.io/v1
 kind: StorageClass
 metadata:
     name: rook-ceph-block
+    labels:
```

Comment:
A similar change would be needed to the CephFS storageClass from file test/addons/rook-cephfs/kustomization.yaml ?